packages/lodestar/src/eth1/utils/depositContract.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @module eth1
+ */
+
+import {ethers} from "ethers";
+import {fromHexString} from "@chainsafe/ssz";
+import {DepositEvent} from "@chainsafe/lodestar-types";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+
+const depositEventFragment =
+  "event DepositEvent(bytes pubkey, bytes withdrawal_credentials, bytes amount, bytes signature, bytes index)";
+
+const depositContractInterface = new ethers.utils.Interface([depositEventFragment]);
```

Comment:
If we're just using ethers for this, we might just use the subpackage: https://github.com/ethers-io/ethers.js/tree/master/packages/abi